src/TestTargets/Spin/Spin.cs

```diff
@@ -9,6 +9,7 @@ private static void Main(string[] args)
             throw new Exception();
 
         Console.WriteLine();
-        Thread.SpinWait(int.MaxValue);
+
+        while (true) ;
```

Comment:
I would prefer a `while (true) Thread.Sleep(1000);` so we don't peg the CPU at max while waiting on the debugger to attach.